src/foam/comics/DAOControllerView.js

```diff
@@ -212,6 +213,15 @@ foam.CLASS({
           .end());
 
       this.add(this.data.border);
+      if (this.isIframe()) this.tag(this.IFrameTopNavigation);
+    },
+
+    function isIframe () {
+      try {
+        return window.self !== window.top;
```

Comment:
Better to import 'window' then do this.window. Makes sure that components can work with multiple windows.